tasks/main.yml

```
@@ -1,49 +1,93 @@
 ---
-# tasks file for slc_ansible_role_maildev
-- name: Check if we are in a flying circus VM
-  ansible.builtin.command: which fc-manage
-  register: is_flyingcircus
-  failed_when:
-    - is_flyingcircus.rc not in [0, 1]
-  changed_when:
-    - False
+- name: Flying Circus detection
+  block:
+    - name: Check if running in a Flying Circus VM
+      ansible.builtin.command: which fc-manage
+      register: fc_check
+      ignore_errors: true
+      failed_when: false
+      changed_when: false
+    - set_fact:
+        is_flyingcircus: "{{ (fc_check.rc == 0) | bool }}"
   tags:
-   - maildev
-   - always
+    - maildev
+    - always
+
+- name: Cleanup legacy maildev installation
+  block:
+    - name: Check for legacy $HOME/maildev directory
+      ansible.builtin.stat:
+        path: "{{ ansible_env.HOME }}/maildev"
```

```suggestion
        path: "{{ maildev_installation_folder }}"
```
Also apply that to the rest of the tasks.